recipes/scikit-odes/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "scikits-odes" %}
+{% set pypiname = "scikits.odes" %}
+{% set version = "2.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: cb5831a26d6a1325d24fc0800e2232195fe09c369d6e754d2f43276882156e4d
+
+build:
+  number: 0
+  # NOTE : There is a pyproject.yml file that seems to force wheels to be built
+  # so remove it.
+  script:
+    - "/bin/rm -f pyproject.toml"
```

Comment:
This seems concerning, does conda not support pyproject.toml?

Reply:
I and others have done this on other feedstocks for conda forge. The PEP 517 change and pip's new behavior regarding it is a bit annoying. We don't build wheels in feedstocks and this is a way to disable that. I had trouble with the `--no-usepep-517` pip flag and `-no-binary` was confusing too. This worked.

Reply:
> This seems concerning, does conda not support pyproject.toml?

I think the answer to that is "not really", because the build is defined in the meta.yaml, build.sh, and build.bat files. `pyproject.toml` is redudant and conda doesn't read it to try to setup the build environment.